Trying to install

I tried cloning and installing...

```
$ git clone git@github.com:BioNGFF/biongff-viewer.git
Cloning into 'biongff-viewer'...
remote: Enumerating objects: 97, done.
remote: Counting objects: 100% (97/97), done.
remote: Compressing objects: 100% (67/67), done.
remote: Total 97 (delta 25), reused 94 (delta 22), pack-reused 0 (from 0)
Receiving objects: 100% (97/97), 216.71 KiB | 964.00 KiB/s, done.
Resolving deltas: 100% (25/25), done.

$ cd biongff-viewer/
$ pnpm i
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing pnpm@9.5.0+sha512.140036830124618d624a2187b50d04289d5a087f326c9edfc0ccd733d76c4f52c3a313d4fc148794a2a9d81553016004e6742e8cf850670268a7387fc220c903.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager

Scope: all 2 workspace projects
/Users/wmoore/Desktop/BIONGFF/biongff-viewer/viewer:
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In viewer: "@hms-dbmi/vizarr@workspace:*" is in the dependencies but no package named "@hms-dbmi/vizarr" is present in the workspace

This error happened while installing a direct dependency of /Users/wmoore/Desktop/BIONGFF/biongff-viewer/viewer

Packages found in the workspace: 
```

Are there some other instructions needed?

--- a/viewer/src/hooks.js
+++ b/viewer/src/hooks.js
@@ -16,17 +16,13 @@ export const useSourceData = (config) => {
         const metadataUrl = `${base}${base.slice(-1) === '/' ? '' : '/'}.zmetadata`;
 
         let metadata = null;
-        let useMetadataFile = false;
-
         try {
           const response = await fetch(metadataUrl);
           if (
             response.ok &&
             response.headers.get('content-type')?.includes('application/json')
           ) {
-            const json = await response.json();
-            metadata = json.metadata;
-            useMetadataFile = true;
+            metadata = await response.json().metadata;
           }
         } catch (e) {
           console.warn(
@@ -38,14 +34,23 @@ export const useSourceData = (config) => {
         const node = await open(store, { kind: 'group' });
 
         if (isBioformats2Raw(node)) {
+          // https://ngff.openmicroscopy.org/0.4/#bf2raw-details
           if (node.attrs['bioformats2raw.layout'] !== 3) {
             setError(new Error('Unsupported bioformats2raw layout'));
             return;
           }
           let series;
-          if (useMetadataFile && metadata && 'OME/.zattrs' in metadata) {
-            const ome = await open(node.resolve('OME'), { kind: 'group' });
-            series = ome.attrs.series;
+          if (metadata) {
+            if ('OME/.zattrs' in metadata) {
+              const ome = await open(node.resolve('OME'), { kind: 'group' });
+              series = ome.attrs.series;
+            } else {
+              const multiscaleKeys = Object.keys(metadata).filter(
+                (key) =>
+                  key.endsWith('/.zattrs') && 'multiscales' in metadata[key],
+              );
+              series = multiscaleKeys.map((key) => key.split('/')[0]);
+            }
           } else {
             const ome = await open(node.resolve('OME'), { kind: 'group' });
             series = ome.attrs?.series || ['0'];

